examples/tic-tac-toe/ui/src/components/NewMultiSigGame.tsx

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useCurrentAccount } from '@mysten/dapp-kit';
+import { PublicKey } from '@mysten/sui/cryptography';
+import { fromB64, toB64 } from '@mysten/sui/utils';
+import { publicKeyFromRawBytes } from '@mysten/sui/verify';
+import { ExclamationTriangleIcon } from '@radix-ui/react-icons';
+import { Box, Button, Em, Flex, Separator, Text, TextField } from '@radix-ui/themes';
+import { ComputedField } from 'components/ComputedField';
+import { useExecutor } from 'hooks/useExecutor';
+import { useTransactions } from 'hooks/useTransactions';
+import { ReactElement, useState } from 'react';
+
+/**
+ * Form for creating a new multi-sig game.
+ */
+export function NewMultiSigGame() {
+	// SAFETY: <App /> tests that a package exists, so Transactions
+	// builder should be available.
+	const tx = useTransactions()!!;
+	const signAndExecute = useExecutor();
+
+	const { address, publicKey: bytes } = useCurrentAccount() || {};
+	const [opponent, setOpponent] = useState<PublicKey | null>(null);
+
+	const publicKey = bytes && publicKeyFromRawBytes('ED25519', bytes);
```

Comment:
Here `useCurrentAccount` has returned some bytes for the public key, and they seem to be raw -- how would a wallet return the public key when the signature scheme was not ED25519?

Reply:
the walletStandard doesn't expose enough information to make this work.  Generally dapps are built around addresses, not public keys, and some wallet accounts won't return meaningful publicKey bytes (eg stashed).

That being said, you theoretically could try each supported schema, and see if it produces a pubKey that has the same address as the account

Reply:
Would it be feasible to have the wallet standard return a public key in "SuiBytes" form? (I believe it's just an enum, so it's the raw bytes prefixed with a variant index), or add an optional signature scheme field, so I could write something like:

```
const { address, publicKey: bytes, keySignature } = useCurrentAccount() || {};
const publicKey = keySignature && bytes && publicKeyFromRawBytes(keySignature, bytes);
```